signup.css

```diff
@@ -114,6 +114,17 @@
 .forgot-password{
     padding: 0;
 }
+.logout-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 80vh;
+}
+.logout-btn-container{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
 @media only screen and (max-width: 1024px){
     .authentication{
         grid-template-columns: 100%;
```

Comment:
You can use em here. Rest of the code looks good. Just a suggestion for logout page there are too much primary button, if possible also use a secondary button, not  necessary though.